CHANGES.md

```diff
@@ -70,6 +70,21 @@
       those docker compose files as well so that you aren't bombarded by deprecation warnings whenever you start
       the birdhouse stack.  
 
+## Fixes
+
+  - Fix bug where compose directory can't be found in `bin/birdhouse` script
+
+    The `COMPOSE_DIR` variable cannnot be discovered properly if:
+    
+    - the `bin/birdhouse` script is called from with the `configs --print-config-command` options.
```

Comment:
So `bin/birdhouse` recursively calls itself, is that why this bug happen?  Is that intended or an accident?

Reply:
No the --print-config-command option just prints to stdout. There's nothing recursive here.

Reply:
Reading "the `bin/birdhouse` script is called from with the `configs --print-config-command` options" somehow I understood
* the `configs` command is from `bin/birdhouse`
* that `configs` command calls `bin/birdhouse`
* so it sounds like `bin/birdhouse` calls itself recursively

Maybe rephrase to make it clearer?